tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -541,6 +556,24 @@ class ScopedFilterDescriptor {
   SE_DISALLOW_COPY_AND_ASSIGN(ScopedFilterDescriptor);
 };
 
+// A helper class to decide whether to enable the TENSOR_OP_MATH math type
+template <bool DefaultFlag>
+class TensorOpMath {
+ public:
+  static bool IsEnabled() {
+    static bool is_enabled = IsEnabledImpl();
+    return is_enabled;
+  }
+
+ private:
+  static bool IsEnabledImpl() {
+    using tensorflow::ReadBoolFromEnvVar;
```

Comment:
This "using" statement doesn't really save us anything.  Recommend not doing it this way, especially because this looks like an ADL idiom, https://stackoverflow.com/questions/28130671/how-does-using-stdswap-enable-adl.

Reply:
Fair enough.